src/modules/sensors/pwm_params_main.c

```diff
@@ -496,7 +496,7 @@ PARAM_DEFINE_INT32(PWM_MAIN_FAIL8, -1);
  * @unit us
  * @group PWM Outputs
  */
-PARAM_DEFINE_INT32(PWM_MAIN_DIS1, -1);
+PARAM_DEFINE_INT32(PWM_MAIN_DIS1, 900);	//-1);
```

Comment:
These should stay to -1 so that all the parameters by default take the value from the PWM_DISARMED parameter which is 900us by default.